src/pages/certificates/filters/filters.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import ContainerFilters from '../../../components/container-filters';
+import ComponentFilterClear from '../../../components/filters-components/filter-clear';
+import ComponentFilterSearch from '../../../components/filters-components/filter-search';
+import ComponentFilterMultiplePicker from '../../../components/filters-components/filter-multiple-picker';
+import buttonTitles from '../../../configs/button-titles';
+
+const availablePrice = [
+  {
+    value: '500',
+    label: '500'
+  },
+  {
+    value: '1000',
+    label: '1000'
+  },
+  {
+    value: '1500',
+    label: '1500'
+  }
+];
+const availableStatus = [
+  {
+    value: '0',
+    label: 'Активний'
+  },
+  {
+    value: '1',
+    label: 'Використаний'
+  },
+  {
+    value: '2',
+    label: 'Прострочений'
+  }
+];
```

Comment:
Pending one is missed. Pending is suppouse to be state of certificate that have a date of activation in the near future.

Reply:
This filters will be removed.